_format/1.1/index.md

```diff
@@ -190,8 +197,18 @@ Here's how an article (i.e. a resource of type "articles") might appear in a doc
 
 #### <a href="#document-resource-object-identification" id="document-resource-object-identification" class="headerlink"></a> Identification
 
-Every [resource object][resource objects] **MUST** contain an `id` member and a `type` member.
-The values of the `id` and `type` members **MUST** be strings.
+As noted above, every [resource object][resource objects] **MUST** contain a
```

Comment:
This portion of the changes actually now lives in #1244, but I think you're keeping it in this PR for now to have a single PR to review everything as one whole?

Reply:
Yeah, I figured portions of this PR would not make sense without basing it on #1244.